src/Akka.Streams.Kafka/Helpers/CommitRefreshing.cs

```diff
@@ -25,7 +27,7 @@ internal static class CommitRefreshing
     {
         public static ICommitRefreshing<K, V> Create<K, V>(TimeSpan commitRefreshInterval)
         {
-            if (commitRefreshInterval == TimeSpan.MaxValue)
+            if (commitRefreshInterval == Timeout.InfiniteTimeSpan)
```

Comment:
Huh, didn't know that HOCON parser uses this... Actually, did not check that out. Thanks!